db/migrate/20160224155909_add_registered_at_to_subscription_facet.rb

```diff
@@ -0,0 +1,9 @@
+class AddRegisteredAtToSubscriptionFacet < ActiveRecord::Migration
+  def up
+    add_column(:katello_subscription_facets, :registered_at, :datetime, :null => false, :default => Time.now)
```

Comment:
Yeah i think that is acceptable, as the user can always re-run the rake task to 'fix' the data.